src/services/communications/Ethernet_packets.py

```diff
@@ -0,0 +1,52 @@
+import struct
+
+
+class Packets:
+    def __init__(self,*packets): 
```

Comment:
this function should get a dict where the keys are the packet ids and the values are a list of the types of each variable in order, something like:
```python
{
    1: ["float32", "float32"],
    2: [],
    3: ["uint8", "bool", "int8"],
}
```

Reply:
this will also make the logic inside the function that much clearer

Reply:
I thought I was going to received diferents packets with id and data_typed and I had to create the dict thats why I created the way you wrote it.  Is not that way

Reply:
?

Reply:
I think it is easier both for you and then when we integrate with the rest of the code to receive the dict as I told you, with a dict from id to list of types